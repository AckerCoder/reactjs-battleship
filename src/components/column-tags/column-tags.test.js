import React from "react";
import { shallow } from "enzyme";
import ColumnTags from "./column-tags.component";

it("Expect to Match Snapshot", () => {
  expect(shallow(<ColumnTags />)).toMatchSnapshot();
});

it("Expect to render CustomInput Component", () => {
  expect(shallow(<ColumnTags />).length).toEqual(1);
});
