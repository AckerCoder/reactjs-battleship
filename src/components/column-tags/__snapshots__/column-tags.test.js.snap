// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect to Match Snapshot 1`] = `
<div
  className="column-tags"
>
  <div
    className="column-tag"
  >
    1
  </div>
  <div
    className="column-tag"
  >
    2
  </div>
  <div
    className="column-tag"
  >
    3
  </div>
  <div
    className="column-tag"
  >
    4
  </div>
  <div
    className="column-tag"
  >
    5
  </div>
  <div
    className="column-tag"
  >
    6
  </div>
  <div
    className="column-tag"
  >
    7
  </div>
  <div
    className="column-tag"
  >
    8
  </div>
  <div
    className="column-tag"
  >
    9
  </div>
  <div
    className="column-tag"
  >
    10
  </div>
</div>
`;
